Dockerfile

```diff
@@ -37,3 +47,28 @@ RUN mix phx.digest
 
 WORKDIR /root
 RUN mix release --verbose
+
+# Copy the release files from the first stage in order to proceed to the second stage
+ADD . rel/site
```

Comment:
Is this instruction correct, @paulswartz ?

Reply:
This was already added on line 24/34, so I don't think you need it here.